marray/tests/test_marray.py

```diff
@@ -997,7 +1194,7 @@ def test_set(f_name, dtype, xp, seed=None):
     res = f_mxp(x)
 
     data[mask] = sentinel
-    ref = xp.unique_all(data)
+    ref = np.unique_all(np.asarray(data))
```

Comment:
Conceptually, the point is that we want to use NumPy's behavior as the reference, since Torch doesn't have `unique_all`. 

Won't work as-written when CuPy is the backend, but that can be addressed in gh-112.